src/main/java/core/basesyntax/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {

```

Comment:
The class `ReturnOperation` directly modifies the `Storage` which can lead to tight coupling. Consider designing this class to use a service for interacting with the `Storage` to adhere to the SOLID principles.